src/main/java/org/mitre/synthea/export/BB2RIFExporter.java

```diff
@@ -2617,19 +2697,12 @@ private long exportHome(Person person, long startTime, long stopTime) throws IOE
           fieldValues.put(HHA.PRNCPAL_DGNS_CD, icdCode);
         }
       }
-
       // PTNT_DSCHRG_STUS_CD: 1=home, 2=transfer, 3=SNF, 20=died, 30=still here
-      String dischargeStatus = null;
-      if (encounter.ended) {
-        dischargeStatus = "1"; // TODO 2=transfer if the next encounter is also inpatient
-      } else {
-        dischargeStatus = "30"; // the patient is still here
-      }
```

Comment:
Looks like you got rid of this? Is that because you never saw it in the data? It should be possible.

Reply:
I never saw it in the exports I ran. If we break up service intervals (one long SNF encounter, multiple consecutive HHA encounters) into multiple claims then that code would be used for non-final claims.